src/v2/Apps/Categories/Components/StickyNav.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Link,
+  Pill,
+  Spacer,
+  themeProps,
+  Swiper,
+  SwiperCell,
+  SwiperRail,
+  SwiperCellProps,
+  SwiperRailProps,
+  Flex,
+} from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { useNavBarHeight } from "v2/Components/NavBar/useNavBarHeight"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+import { scrollIntoView } from "v2/Utils/scrollHelpers"
+import { StickyNav_geneFamiliesConnection } from "v2/__generated__/StickyNav_geneFamiliesConnection.graphql"
+interface StickyNavProps {
+  geneFamiliesConnection: StickyNav_geneFamiliesConnection
+}
+
+const StickyNav: React.FC<StickyNavProps> = props => {
+  const { geneFamiliesConnection } = props
+  const geneFamilies = extractNodes(geneFamiliesConnection)
+  const { mobile, desktop } = useNavBarHeight()
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
```

Comment:
We'll want to avoid this as it's not SSR friendly

Reply:
Good to know, thanks for calling out! I replaced it with [this](https://github.com/artsy/force/blob/62327e17f0397434f9c438c3e3064d0e2fb87796/src/v2/Apps/Categories/CategoriesApp.tsx#L20) and [this](https://github.com/artsy/force/blob/62327e17f0397434f9c438c3e3064d0e2fb87796/src/v2/Apps/Categories/CategoriesApp.tsx#L35-L46) on the parent - so the parent is now using `<Media at...>` to determine whether it should pass down the mobile or desktop nav bar height. It's a smidge messy, but seems OK 🤷